sentry-samples/sentry-samples-spring-boot-webflux-jakarta/src/main/resources/application.properties

```diff
@@ -8,3 +8,4 @@ sentry.max-breadcrumbs=150
 sentry.logging.minimum-event-level=info
 sentry.logging.minimum-breadcrumb-level=debug
 sentry.reactive.thread-local-accessor-enabled=true
+sentry.enable-tracing=false
```

Comment:
why false here?

Reply:
Slipped in while testing; I changed it to `true`